src/app/space/app-launcher/import-app/import-app.component.ts

```diff
@@ -66,4 +68,17 @@ export class ImportAppComponent implements OnDestroy, OnInit {
   complete(): void {
     this.router.navigate(['/', this.loggedInUser.attributes.username, this.currentSpace.attributes.name]);
   }
+
+  addQuery() {
```

Comment:
This method looks exactly same as the one in create app component. Can it be reused somehow? Also, missing return type.